test/functional/data/invalid_txs.py

```diff
@@ -114,15 +115,16 @@ def get_tx(self):
 
 # The following check prevents exploit of lack of merkle
 # tree depth commitment (CVE-2017-12842)
-class SizeTooSmall(BadTxTemplate):
-    reject_reason = "tx-size-small"
+class BadNonWitSize(BadTxTemplate):
+    reject_reason = "tx-bad-nonwit-size"
     expect_disconnect = False
     valid_in_block = True
 
     def get_tx(self):
         tx = CTransaction()
         tx.vin.append(self.valid_txin)
-        tx.vout.append(CTxOut(0, CScript([OP_TRUE])))
+        tx.vout.append(CTxOut(0, CScript([OP_RETURN] + ([OP_0] * (INVALID_SPK_LEN - 1)))))
```

Comment:
why not NONSTANDARD_OP_RETURN_SCRIPT?

Reply:
done

Reply:
You'll need to remove the unused imports now